packages/@sanity/core/src/actions/build/buildStaticAssets.js

```diff
@@ -96,7 +97,7 @@ export default async (args, context) => {
     const indexStart = Date.now()
     spin = output.spinner('Building index document').start()
     const doc = await getDocumentElement(
-      {...compilationConfig, hashes: chunkMap, project: config.get('project')},
```

Comment:
Regarding adding the ability to specify basePath as an option in the cli, I was thinking of using it like this:

```javascript
    const doc = await getDocumentElement(
      {...compilationConfig, hashes: chunkMap},
      {
        basePath: flags.basePath || (project && project.basePath) || '',
        scripts: ['vendor.bundle.js', 'app.bundle.js'].map(asset => {
          const assetPath = absoluteMatch.test(asset) ? asset : `js/${asset}`
          return {
            path: assetPath,
            hash: chunkMap[assetPath] || chunkMap[asset]
          }
        })
      }
    )
`
where somewhere before you do `const project = config.get('project')`